virtcontainers/cgroups.go

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2018 Huawei Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package virtcontainers
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/containerd/cgroups"
+	"github.com/kata-containers/runtime/virtcontainers/pkg/annotations"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+const (
+	vcpuGroupName       = "vcpu"
+	defaultCgroupParent = "/vc"
```

Comment:
@WeiZhang555 Maybe add this under `kata` to not cause any conflicts?

Reply:
This sounds good.